packages/babel-traverse/src/path/evaluation.js

```diff
@@ -112,25 +112,19 @@ export function evaluate(): { confident: boolean; value: any } {
     }
 
     if (path.isTemplateLiteral()) {
-      let str = "";
-
-      let i = 0;
-      const exprs = path.get("expressions");
-
-      for (const elem of (node.quasis: Array<Object>)) {
-        // not confident, evaluated an expression we don't like
-        if (!confident) break;
+      return evaluateQuasis(path, node.quasis);
+    }
 
-        // add on cooked element
-        str += elem.value.cooked;
+    if (path.isTaggedTemplateExpression()) {
```

Comment:
Ah, of course. I was still thinking of the code before I added in the destructuring, and I wasn't sure how `path.get` behaves with non-existent paths. Thanks for the explanation.